webapp/src/tutorialCodeValidation.tsx

```diff
@@ -118,6 +124,7 @@ export class ShowValidationMessage extends data.Component<TutorialCodeValidation
         const rulesDefined = (rules != undefined);
         const strictRulePresent = this.props.areStrictRulesPresent;
         return <div>
+            {this.state.visible && <div className="mask" role="region" onClick={this.props.onNoButtonClick}></div>}
```

Comment:
do you need both this mask and the one in tutorial.tsx?

Reply:
Good point, I don't think it's necessary to have it in two places.